seminaire-saint-sulpice-ecole-theologie.csl

```diff
@@ -7,26 +7,30 @@
     <id>http://www.zotero.org/styles/seminaire-saint-sulpice-ecole-theologie</id>
     <link href="http://www.zotero.org/styles/seminaire-saint-sulpice-ecole-theologie" rel="self"/>
     <link href="http://www.zotero.org/styles/campus-adventiste-du-saleve-faculte-adventiste-de-theologie" rel="template"/>
-    <link href="http://www.sulpissy.info/" rel="documentation"/>
+    <link href="http://www.seminairesaintsulpice.fr/" rel="documentation"/>
     <author>
       <name>Augustin CHARTIER</name>
     </author>
     <category citation-format="note"/>
     <category field="theology"/>
     <category field="humanities"/>
-    <updated>2014-05-30T00:36:38+00:00</updated>
+    <updated>2016-06-13T16:02:33+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="fr">
     <terms>
-      <term name="cited"/>
-      <term name="page" form="short">
+        <term name="in">
+            <single>dans</single>
+            <multiple>dans</multiple>
+        </term>
```

Comment:
Can you just use `<term name="in">dans</term>`? There is no need to define singular and plural versions.